guides/source/active_storage_overview.md

```diff
@@ -526,17 +526,21 @@ user.avatar.purge_later
 Linking to Files
 ----------------
 
-Generate a permanent URL for the blob that points to the application. Upon
-access, a redirect to the actual service endpoint is returned. This indirection
-decouples the service URL from the actual one, and allows, for example, mirroring
-attachments in different services for high-availability. The redirection has an
-HTTP expiration of 5 minutes.
+To generate a permanent URL for a blob use [`url_for`][ActionView::RoutingUrlFor].
+This returns a URL with the [`signed_id`][ActiveStorage::Blob] for the blob and
+points to the application.
 
 ```ruby
 url_for(user.avatar)
+# => /rails/active_storage/blobs/:signed_id/my-avatar.png
 ```
 
-WARNING: The links generated by ActiveStorage are hard to guess, but publicly
+When the URL is accessed a redirect to the actual service endpoint is returned.
```

Comment:
Nice - I also tried to make this a bit clearer in https://github.com/rails/rails/pull/42495

Reply:
Lots of activity in the ActiveStorage guides 😄